File: C++_class/account.cpp
#include <iostream>
using namespace std;
class Account{
public:
    string accountHolder;
    string accountNumber;
    int accountBalance =0;
    void creatAccount(string accHolder,string accNumber){
        this-> accountHolder=accHolder;
        this-> accountNumber=accNumber;
        accountBalance=1000;

    }
    void deposite(int depositeAmount){
        accountBalance=accountBalance + depositeAmount;
    }
    bool withdraw(int withdrawAmount){
        if(withdrawAmount>accountBalance){
            return false;
        }else{
            accountBalance=accountBalance-withdrawAmount;
            return true;
        }
    }
    void showData(){
        cout << "_______Current Info________" << endl;
        cout << "Account Holder Name : " << accountHolder << endl;
        cout << "Account Number : " << accountNumber << endl;
        cout << "Account Balance : " << accountBalance << endl;
    }

};

int main(){
    Account account1;
    account1.showData();
    account1.creatAccount("Zwe Kaung","12345");
    account1.showData();
    account1.deposite(20000);
    bool result = account1.withdraw(50000);
    if(result){
        account1.showData();
    }else{
        cout << "Failed to withdraw! " << endl;
    }

    return 0;
}

File: python/Account.py
class Account:
    def __init__(self,name,account_number,initial_amount):
        self.name=name
        self.no=account_number
        self.balance=initial_amount
    def deposite(self,amount):
        self.balance += amount
    def withdrawl(self,amount):
        self.balance -= amount
    def dump(self):
        print('%s , %s balance : %s '%(self.name,self.no,self.balance))
        
        
a1=Account('Ma Ma','0123456',20000)
print(a1.__dict__)
a1.dump()
a1.deposite(50000)
a1.dump()
a1.withdrawl(60000)
a1.dump()